src/interface/src/app/shared/form-message-box/form-message-box.component.scss

```diff
@@ -0,0 +1,50 @@
+:host {
+  .message-box {
+    border-radius: 2px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+    padding: 12px 10px 12px 10px;
+    background-color: rgba(245, 245, 245, 0.4000000059604645);
+    margin-top: 5px;
+    margin-bottom: 1em;
+    font-family: 'Public Sans', 'Roboto', Arial, Helvetica, sans-serif;
+    border: 1px solid black;
+  }
+  .message-box.success {
+    border: 1px solid green;
+  }
+  .message-box.error {
+    border: 1px solid red;
+  }
+
+  :host.message-icon {
+    padding-right: 0.8em;
+    float: left;
+  }
+
+  :host.message-title {
```

Comment:
maybe Im confused by I don't see where this is applied to the host.
That's definitely one way to go, and you'll save one precious wrapper element on your html, but you'll need to hook up [host binding](https://angular.io/api/core/HostBinding) for it to work